.claude/skills/debug/SKILL.md

```
@@ -0,0 +1,75 @@
+# Debug Skill
+
+Systematic debugging workflow.
+
+## Phase 1: Reproduce
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)